### PR TITLE
Add PHP 8 support

### LINKED ISSUE #101: Mark PHP 8 as supported
I've tried the plugin with PHP 8 on many Yii 3 packages and it works well (same as with PHP 7.4). The only thing missing is Composer constraint to be changed to `"php": "^7.4|^8.0"`. Would you accept a pull request?

### PR DESCRIPTION
**Pull request by samdark**
Fixes #101

### CODE REVIEW

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
           - "locked"
         php-version:
           - "7.4"
+          - - "8.0"
```
Comment (Ocramius) [inline .github/workflows/phpunit.yml]: Duplicate `-` here

**Comment by Ocramius** [comment]
@samdark `composer.lock` also needs updating, I think 🤔

**Comment by samdark** [comment]
Here we go.

**Comment by samdark** [comment]
Please tag a release. Would allow us to do less `--ignore-platform-reqs` :)

**Review by Ocramius — APPROVED** [review]
LGTM 👍

**Comment by Ocramius** [comment]
Thanks @samdark! It's released 👍

**Comment by samdark** [comment]
Great. Thanks!

**Pull request merged**